tests/pyincore/analyses/pipelinerestoration/test_pipelinerestoration.py

```diff
@@ -0,0 +1,36 @@
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+
+
+from pyincore import IncoreClient, FragilityService, MappingSet
+from pyincore.analyses.pipelinerestoration import PipelineRestoration
+import pyincore.globals as pyglobals
+
+
+def run_with_base_class():
+    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
+    client.clear_cache()
+    pipeline_restoration = PipelineRestoration(client)
+
+    # shelby county pipelines
+    pipeline_restoration.load_remote_input_dataset("pipeline", "5a284f28c7d30d13bc081d14")
+    pipeline_restoration.load_remote_input_dataset("pipeline_damage", "61f36023c53b3620b6b614c6")
+
+    # Load fragility mapping
+    fragility_service = FragilityService(client)
```

Comment:
This should have been RestorationService, as the analysis doesn't use fragilities. It doesn't impact the output of the analysis, it will just be misleading to the users.